docs/modules/plugins/pages/generic-ui-steps.adoc

```diff
@@ -0,0 +1,68 @@
+= Generic UI Steps
+
+Generic UI steps are used to interact with web applications and mobile applications (on Read Devices, Android Emulators and iOS Simulators) and become available after adding xref:plugins:plugin-web-app.adoc[plugin-web-app] or xref:plugins:plugin-mobile-app.adoc[plugin-mobile-app] plugins as project dependencies.
+
+== Steps
+
+=== Verify elements number
+
+==== *_Info_*
+
+Verifies if the number of elements located by locator matches desired number
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+Then number of elements found by `$locator` is $comparisonRule `$quantity`
+----
+
+==== *_Parameters_*
+
+. `$locator` - refer to xref:parameters:locator.adoc[Locator] for web locators and to xref:plugins:generic-ui-steps.adoc[Generic UI Steps] for mobile locators
```

Comment:
ideally docs for web app should contain link to web locators, but mobile app - mobile locators (without such options like now)